Glideslope.json

```diff
@@ -11604,7 +11604,7 @@
     "elevation": 749, 
     "name": "GS", 
     "longitude": -80.94994654, 
-    "associatedRunwayNumber": "5", 
+    "associatedRunwayNumber": "05", 
```

Comment:
Is the runway named 05 or 5? travis will accept both just make sure if the runway is just 5 then name it as just 5

Reply:
Oh, sorry.
I had same mistake.
If WED showed "05" I can write "05"?